design-documents/graph-ql/coverage/catalog/compare-list.md

```diff
@@ -4,6 +4,12 @@
 * Loginned buyer can access the latest compare list if the list has not been cleared with the last session. 
 * The buyer can request a compare list which has to return as a bare minimum price, link to a product page, name, and SKU.
 
+
+* A Compare list ID is a client-side generated ID.
+* A new compare list could be created by calling `addItemsToCompareList`.
+* If a shopper calls  `addItemsToCompareList` with the new id the previous cart will be retired and replaced with a new one.
```

Comment:
This does not sound right:
1. Why we are talking about cart?
2. If we assume it should be compare list, why/how do we retire another one?